public/app/features/transformers/docs/content.ts

```diff
@@ -98,7 +96,11 @@ export const transformationDocsContent: TransformationDocsContentType = {
     name: 'Concatenate fields',
     getHelperDocs: function () {
       return `
-  Use this transformation to combine all fields from all frames into one result. Consider the following:
+  Use this transformation to combine all fields from all frames into one result.
+  
+  Suppose you have separate queries retrieving temperature and uptime data (Query A) and air quality index and error information (Query B). Applying the concatenate transformation would yield a consolidated data frame with all relevant information in one view.
```

Comment:
Yep, this is called a `template string` or `template literal` in Javascript (when one uses backticks instead of traditional quotes), and it's used to enforce the correct whitespace formatting.